src/Phinx/Db/Adapter/SqlServerAdapter.php

```diff
@@ -224,7 +224,7 @@ public function hasTable(string $tableName): bool
         }
 
         /** @var array<string, mixed> $result */
-        $result = $this->fetchRow(sprintf("SELECT count(*) as [count] FROM information_schema.tables WHERE table_name = '%s';", $tableName));
+        $result = $this->fetchRow(sprintf("SELECT count(*) as [count] FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_NAME = '%s';", $tableName));
```

Comment:
Correct, otherwise it will trigger an error. I also tried to make the tests run on a case sensitive database, but somehow in the workflow environment using a case sensitive collation always causes the server to crash, while it's working fine locally. So far I wasn't able to figure out what the problem is.